.github/workflows/on-merge.yml

```diff
@@ -10,23 +10,33 @@ jobs:
     name: 'Label and Backport'
     runs-on: ubuntu-latest
     if: |
-      github.event.pull_request.merged == true &&
-        (github.event.action == 'closed' ||
-          (github.event.action == 'labeled' &&
-            (github.event.label.name == 'backport:prev-minor' ||
-              github.event.label.name == 'backport:prev-major' ||
-              github.event.label.name == 'backport:current-major' ||
-              github.event.label.name == 'backport:all-open' ||
-              github.event.label.name == 'backport:version' ||
-              github.event.label.name == 'auto-backport')) ||
-          (github.event.action == 'unlabeled' &&
-            github.event.label.name == 'backport:skip' &&
-            (contains(github.event.pull_request.labels.*.name, 'backport:prev-minor') ||
-              contains(github.event.pull_request.labels.*.name, 'backport:prev-major') ||
-              contains(github.event.pull_request.labels.*.name, 'backport:current-major') ||
-              contains(github.event.pull_request.labels.*.name, 'backport:all-open') ||
-              contains(github.event.pull_request.labels.*.name, 'backport:version') ||
-              contains(github.event.pull_request.labels.*.name, 'auto-backport'))))
+      github.event.pull_request.merged == true && (
+        github.event.action == 'closed' || (
+          github.event.action == 'labeled' && (
+            github.event.label.name == 'backport:prev-minor' ||
+            github.event.label.name == 'backport:prev-major' ||
+            github.event.label.name == 'backport:current-major' ||
+            github.event.label.name == 'backport:all-open' ||
+            github.event.label.name == 'backport:version' ||
+            github.event.label.name == 'auto-backport'
+          )
+        ) || (
+          github.event.action == 'labeled' && (
```

Comment:
Is this missing the `contains(github.event.pull_request.labels.*.name, 'backport:version')` condition?

Reply:
Indeed it does, thanks!